lib/rspecq/queue.rb

```diff
@@ -67,6 +67,9 @@ class Queue
 
     STATUS_INITIALIZING = "initializing".freeze
     STATUS_READY = "ready".freeze
+    DEFAULT_CONFIG = {
```

Comment:
[style] Leave a newline above

Reply:
So as we discussed, this is no longer needed since we'll move fail fast to a regular attribute, (i.e. `queue.fail_fast`).